packages/sui-mono/package.json

```diff
@@ -1,15 +1,11 @@
 {
   "name": "@schibstedspain/sui-mono",
-  "version": "1.2.0",
+  "version": "1.5.0",
```

Comment:
Upgrade of more than one minor for testing purpose. I wanted to try publishing with and without build script in "real"